pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import functools
+import subprocess
+from subprocess import STDOUT
+
+import pwndbg.commands
+import pwndbg.which
+
+
+
+class OnlyWithCommand(object):
+    def __init__(self, command):
```

Comment:
We should probably register command name in some global set and check for its existence in the system after pwndbg is initialized. If it is not there - warn the user and disable particular command.

^This can be done in next PR; if we decide not to do this in current PR, please create an issue for it.